models/indicator_consolidation.py

```diff
@@ -103,3 +104,4 @@ def populate_prep(cls, start):
                         },
                         {"value": value, "updated_at": datetime.now()},
                     )
+        db.update("SET timezone='UTC';")
```

Comment:
Why can't we just adjust the timezone in the code? It doesn't look right to do this in the database. What if one of these loops breaks? All the transactions would be registered as having a different timezone. If we had multiple clients talking to the database, this would be detrimental as well.

Reply:
The server is always at UTC. This part of the code does only a **session** update. This means that if a loop breaks it would come back to UTC and it does not affect other clients talking to the database. I have only updated it back to UTC for future implementations using the same session.

Reply:
Is it not clear? Maybe the [documentation of postgres ](https://www.postgresql.org/docs/9.1/sql-set.html)will help:

> The SET command changes run-time configuration parameters. Many of the run-time parameters listed in Chapter 18 can be changed on-the-fly with SET. (But some require superuser privileges to change, and others cannot be changed after server or session start.) **SET only affects the value used by the current session**.
 If SET (or equivalently SET SESSION) is issued within a transaction that is later aborted, the **effects of the SET command disappear when the transaction is rolled back**. Once the surrounding transaction is committed, the effects will persist until the end of the session, unless overridden by another SET.